src/tespy/tools/analyses.py

```diff
@@ -0,0 +1,759 @@
+            will be applied.
+
+        Returns
+        -------
+        tuple
+            Tuple containing the links and node_order for the plotly sankey
+            diagram.
+        """
+        group_data = self.group_data.copy()
+
+        for fkt_group, data in self.group_data.items():
+            group_data[fkt_group] = group_data[fkt_group][
+                data['value'].abs() >= display_thresold]
+
+        self.remove_transit_groups(group_data)
+
+        if len(node_order) == 0:
+            node_order = (
+                ['E_F'] + [b.label for b in self.E_F] +
+                [fkt_group for fkt_group in self.group_overview.index] +
+                [b.label for b in self.internal_busses + self.E_P + self.E_L] +
+                ['E_P', 'E_L', 'E_D'])
+
+        if len(colors) == 0:
+            cmap = cm.get_cmap('Set1')(np.linspace(0.0, 1.0, 10))
+            cmap[:, 0:3] *= 255
+            cmap[:, 3] *= 0.75
+            rgba_list = []
+            for i in range(cmap.shape[0]):
+                rgba_list += [
+                    'rgba(' +
+                    str(cmap[i, 0]) + ', ' +
+                    str(cmap[i, 1]) + ', ' +
+                    str(cmap[i, 2]) + ', ' +
+                    str(cmap[i, 3]) + ')']
+
+            colors = {}
+            colors['E_P'] = rgba_list[0]
+            colors['E_F'] = rgba_list[1]
+            colors['E_D'] = rgba_list[2]
+            colors['E_L'] = rgba_list[3]
+            i = 4
+            for f in self.nw.fluids:
+                colors[f] = rgba_list[i]
+                i += 1
+            for b in self.internal_busses:
+                colors[b.label] = rgba_list[i]
+                i += 1
+
+        links = {
+            'source': [],
+            'target': [],
+            'value': [],
+            'color': []
+        }
+
+        for fkt_group, data in group_data.items():
+            source_id = node_order.index(fkt_group)
+            links['source'] += [source_id for i in range(len(data))]
+            links['target'] += [
+                node_order.index(target) for target in data.index]
+            links['value'] += [
+                data.loc[target, 'value'] for target in data.index]
+            # connection colors
+            for cat in data['cat']:
+                links['color'].append(colors[cat])
+
+        return links, node_order
+
+    def print_results(
+            self, E_D_min=1000, sort_desc=True,
+            busses=True, components=True, connections=True, groups=True,
+            network=True):
+        r"""Print the results of the exergy analysis to prompt.
+
+        - The results are sorted beginning with the component having the
+          biggest exergy destruction by default.
+        - Components with an exergy destruction smaller than 1000 W is not
+          printed to prompt by default.
+
+        Parameters
+        ----------
+        E_D_min : float
+            Minimum exergy destruction to be printed to prompt.
```

Comment:
@juliusmeier, we might want to remove this entry as the user can simply save the DataFrame into his variable space for further processing and there is no necessity to have this in the code. What do you think?

Reply:
Good point, I agree.